opencage_geocoder.go

```diff
@@ -115,8 +134,13 @@ func (g *OpenCageGeocoder) ReverseGeocode(p *Point) (string, error) {
 // Return sthe first address in the passed in byte array.
 func (g *OpenCageGeocoder) extractAddressFromResponse(data []byte) string {
 	res := &opencageGeocodeResponse{}
-	json.Unmarshal(data, &res)
+	if err :=  json.Unmarshal(data, &res) ; err != nil {
+        panic(err)
+    }
```

Comment:
Ha, yeah silly of me to add this panic while removing another. Thanks for catching that! I'll yank that out and resubmit.